### PR TITLE
docs(component-store): add alert about NgRx Signals

### PR DESCRIPTION
**Pull request by markostanimirovic**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/ngrx/platform/blob/main/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Documentation has been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[x] Documentation content changes
[ ] Other... Please describe:
```

## Does this PR introduce a breaking change?

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: projects/ngrx.io/content/guide/component-store/comparison.md [hunk 1]
```diff
@@ -1,3 +1,12 @@
+<div class="alert is-helpful">
+
+**<a href="/guide/signals"><b>NgRx Signals</b></a> is the new default.**
+
+The NgRx team recommends using `@ngrx/signals` library for local state management in Angular.
```
Comment (timdeschryver) [inline projects/ngrx.io/content/guide/component-store/comparison.md:5]:
```suggestion
The NgRx team recommends using the `@ngrx/signals` library for local state management in Angular.
```

File: projects/ngrx.io/content/guide/component-store/effect.md [hunk 2]
```diff
@@ -1,3 +1,12 @@
+<div class="alert is-helpful">
+
+**<a href="/guide/signals"><b>NgRx Signals</b></a> is the new default.**
+
+The NgRx team recommends using `@ngrx/signals` library for local state management in Angular.
```
Comment (timdeschryver) [inline projects/ngrx.io/content/guide/component-store/effect.md:5]:
```suggestion
The NgRx team recommends using the `@ngrx/signals` library for local state management in Angular.
```

File: projects/ngrx.io/content/guide/component-store/index.md [hunk 3]
```diff
@@ -1,3 +1,12 @@
+<div class="alert is-helpful">
+
+**<a href="/guide/signals"><b>NgRx Signals</b></a> is the new default.**
+
+The NgRx team recommends using `@ngrx/signals` library for local state management in Angular.
```
Comment (timdeschryver) [inline projects/ngrx.io/content/guide/component-store/index.md:5]:
```suggestion
The NgRx team recommends using the `@ngrx/signals` library for local state management in Angular.
```

File: projects/ngrx.io/content/guide/component-store/initialization.md [hunk 4]
```diff
@@ -1,3 +1,12 @@
+<div class="alert is-helpful">
+
+**<a href="/guide/signals"><b>NgRx Signals</b></a> is the new default.**
+
+The NgRx team recommends using `@ngrx/signals` library for local state management in Angular.
```
Comment (timdeschryver) [inline projects/ngrx.io/content/guide/component-store/initialization.md:5]:
```suggestion
The NgRx team recommends using the `@ngrx/signals` library for local state management in Angular.
```

File: projects/ngrx.io/content/guide/component-store/install.md [hunk 5]
```diff
@@ -1,3 +1,12 @@
+<div class="alert is-helpful">
+
+**<a href="/guide/signals"><b>NgRx Signals</b></a> is the new default.**
+
+The NgRx team recommends using `@ngrx/signals` library for local state management in Angular.
```
Comment (timdeschryver) [inline projects/ngrx.io/content/guide/component-store/install.md:5]:
```suggestion
The NgRx team recommends using the `@ngrx/signals` library for local state management in Angular.
```

File: projects/ngrx.io/content/guide/component-store/lifecycle.md [hunk 6]
```diff
@@ -1,3 +1,12 @@
+<div class="alert is-helpful">
+
+**<a href="/guide/signals"><b>NgRx Signals</b></a> is the new default.**
+
+The NgRx team recommends using `@ngrx/signals` library for local state management in Angular.
```
Comment (timdeschryver) [inline projects/ngrx.io/content/guide/component-store/lifecycle.md:5]:
```suggestion
The NgRx team recommends using the `@ngrx/signals` library for local state management in Angular.
```

File: projects/ngrx.io/content/guide/component-store/read.md [hunk 7]
```diff
@@ -1,3 +1,12 @@
+<div class="alert is-helpful">
+
+**<a href="/guide/signals"><b>NgRx Signals</b></a> is the new default.**
+
+The NgRx team recommends using `@ngrx/signals` library for local state management in Angular.
```
Comment (timdeschryver) [inline projects/ngrx.io/content/guide/component-store/read.md:5]:
```suggestion
The NgRx team recommends using the `@ngrx/signals` library for local state management in Angular.
```

File: projects/ngrx.io/content/guide/component-store/usage.md [hunk 8]
```diff
@@ -1,3 +1,12 @@
+<div class="alert is-helpful">
+
+**<a href="/guide/signals"><b>NgRx Signals</b></a> is the new default.**
+
+The NgRx team recommends using `@ngrx/signals` library for local state management in Angular.
```
Comment (timdeschryver) [inline projects/ngrx.io/content/guide/component-store/usage.md:5]:
```suggestion
The NgRx team recommends using the `@ngrx/signals` library for local state management in Angular.
```

File: projects/ngrx.io/content/guide/component-store/write.md [hunk 9]
```diff
@@ -1,3 +1,12 @@
+<div class="alert is-helpful">
+
+**<a href="/guide/signals"><b>NgRx Signals</b></a> is the new default.**
+
+The NgRx team recommends using `@ngrx/signals` library for local state management in Angular.
```
Comment (timdeschryver) [inline projects/ngrx.io/content/guide/component-store/write.md:5]:
```suggestion
The NgRx team recommends using the `@ngrx/signals` library for local state management in Angular.
```

**Review by timdeschryver — APPROVED** [review]
Should we also create a page (in another PR) why we encourage it?

File: projects/ngrx.io/content/guide/component-store/comparison.md (hunk 1 above)
Comment (brandonroberts) [reply projects/ngrx.io/content/guide/component-store/comparison.md:5]: Should we encourage migrating existing ComponentStores to SignalStore?

Comment (rainerhahnekamp) [reply projects/ngrx.io/content/guide/component-store/comparison.md:5]: Yes… and just an idea: I would even suggest adding a note that for new projects, `@ngrx/signals` should be the recommended choice for global state management. Many applications out there are not so complicated to require Redux for global management.

**Review by brandonroberts — APPROVED** [review]

**Pull request merged**